extensions/ql-vscode/src/remote-queries/remote-queries-markdown-generation.ts

```diff
@@ -25,8 +34,37 @@ export function generateMarkdown(query: RemoteQuery, analysesResults: AnalysisRe
     }
     files.push(lines);
   }
+  files.push(summaryLines);
```

Comment:
Would we want the summary be the first file being returned
to be the first file so it shows at the top? If so we can `return [summaryLines, ...files]` instead of pushing and returning.

Reply:
Makes sense, thank you!